src/view/dataElements/eventMergeId.styl

```diff
@@ -10,7 +10,8 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-const createEcid = require("./createEcid");
-const instanceManager = require("../../instanceManager/index");
+@import "../global";
 
-module.exports = createEcid({ instanceManager, turbine });
+.EventMergeId-description {
+  font-size: 18px;
+}
```

Comment:
This diff is misleading since this file has nothing to do with `ecid/index.js`. I removed the getEcid stuff until we figure out how we're actually going to handle ECIDs (and it sounds like the command name will be changing too). The ECID stuff we did have was not actually being used (it wasn't referenced from `extension.json`).